portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I was born in Japan!', 'Orange is my favorite color!', 'I play volleyball！', 'I am allergic to cats and dogs!'];
```

Comment:
I think you'll need to merge master into this branch or rebase this branch onto master in order for this file to appear deleted like it actually is.

Reply:
I think I merged the master into this branch

Reply:
Hmm it looks like this is still out of date, since you deleted this file in a previous PR

I talked with Aleksey about this, and we think you'll need to run "git pull" first before merging master, since you'll need to pull the latest changes from your remote repo into your local repo before merging master. 

Can you try doing this and see if that helps? If not, we can try video chat tomorrow to resolve this.